sdk-rs/src/constants.rs

```diff
@@ -0,0 +1,163 @@
+use std::sync::OnceLock;
+use std::str::FromStr;
+
+use drift_program::state::{perp_market::PerpMarket, spot_market::SpotMarket};
+pub use drift_program::{
+    math::constants::{
+        BASE_PRECISION_U64 as BASE_PRECISION, PRICE_PRECISION,
+        QUOTE_PRECISION_U64 as QUOTE_PRECISION, SPOT_BALANCE_PRECISION,
+    },
+    ID as PROGRAM_ID,
+};
+use solana_sdk::{address_lookup_table_account::AddressLookupTableAccount, pubkey::Pubkey};
+use substreams_solana_macro::b58;
+
+use crate::types::Context;
+
+static STATE_ACCOUNT: OnceLock<Pubkey> = OnceLock::new();
+
+lazy_static::lazy_static! {
+    pub static ref TOKEN_PROGRAM_ID: Pubkey = Pubkey::from_str("TokenkegQfeZyiNwAJbNbGKPFXCWuBvf9Ss623VQ5DA").unwrap();
+}
```

Comment:
Suggestion: you can use `solana_sdk::pubkey!()` macro to create Pubkey constants without the need for `lazy_static`
```suggestion
pub const TOKEN_PROGRAM_ID: Pubkey = solana_sdk::pubkey!("TokenkegQfeZyiNwAJbNbGKPFXCWuBvf9Ss623VQ5DA");
```

Reply:
Hi, I tried pubkey! but it says it's expecting an array